### PR TITLE
Fix staking check

### PR DESCRIPTION
**Pull request by jmoreira-valory**
Fix staking check

### CODE REVIEW

**Review by mariapiamo — APPROVED** [review]

File: scripts/staking.py [hunk 1]
```diff
@@ -144,33 +182,7 @@ def unstake_everest(
...
-            )
-            last_ts = next_ts - liveness_period
-            now = time.time()
-
-            minimum_staking_duration = get_min_staking_duration(
-                ledger_api, args.staking_contract_address
-            )
-            available_rewards = get_available_rewards(
-                ledger_api, args.staking_contract_address
-            )
```
Comment (mariapiamo) [inline scripts/staking.py:163]: not sure why lines 143-163 are removed

Comment (jmoreira-valory) [reply scripts/staking.py:163]: They are not removed, they are moved above to collect this information, which is constant for all the workflow below.

File: scripts/staking.py [hunk 2]
```diff
@@ -128,6 +146,26 @@ def unstake_everest(
...
+        )
+        last_ts = next_ts - liveness_period
+        now = time.time()
+
+        minimum_staking_duration = get_min_staking_duration(
+            ledger_api, args.staking_contract_address
+        )
+        available_rewards = get_available_rewards(
+            ledger_api, args.staking_contract_address
+        )
```
Comment (jmoreira-valory) [inline scripts/staking.py:167]: @mariapiamo they are moved here.

Comment (mariapiamo) [reply scripts/staking.py:167]: sorry, thanks!

**Review by mariapiamo — APPROVED** [review]

**Review by 0xArdi — APPROVED** [review]

**Pull request merged**